packages/nimble-components/package.json

```diff
@@ -99,7 +99,8 @@
     "@storybook/theming": "^7.0.4",
     "@types/jasmine": "^4.3.1",
     "@types/webpack-env": "^1.15.2",
-    "babel-loader": "^ 9.1.2",
+    "babel-loader": "^9.1.2",
+    "chromatic": "^6.17.3",
```

Comment:
That ended up pulling a lot of node_module dependencies in the lock file for what is ultimately a 5 line script:
https://github.com/chromaui/chromatic-cli/blob/24a3816e956f724e335fe649bb8fa77fda308b40/isChromatic.mjs#L3

Maybe that's okay? Then we don't have to maintain it.

I think it's probably low-risk of changing as it gets compiled into folks built stories so a breaking change would break existing stories. Think we could vendor it under `utilities/tests/` and link to the original source. Don't feel super strongly either way. If we think we may need more from it eventually maybe it's worth it.

Reply:
Great point. I ended up duplicating it and removing the dependency.

Ironically the ~75 new dependencies were coming from [snyk/nodejs-lockfile-parser](https://github.com/snyk/nodejs-lockfile-parser), "a small utility package" from an organization that "helps you find, fix and monitor for known vulnerabilities in your dependencies". So hopefully at least none of the 75 deps had known vulnerabilities!